### PR TITLE
Arxivops 1432 code of conduct enforcement

### PR DESCRIPTION
**Pull request by jimentwood**

### CODE REVIEW

File: help/policies/code_of_conduct_enforcement.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# arXiv Code of Conduct Enforcement Options
+
+arXiv is a platform where researchers engage with one another to participate in science, software development, and scholarly communications. It is also a community.
+
+Our valued community of authors, readers, moderators, advisory board members, staff, collaborators, and supporters expects and deserves a respectful environment, which is why we adhere to a [Code of Conduct](code_of_conduct), [moderation standards]( /help/moderation), and other policies.
```
Comment (afromme) [inline help/policies/code_of_conduct_enforcement.md:5]: Suggestion: change "advisory board members" to "advisors"

File: help/policies/code_of_conduct_enforcement.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+There are several actions that if taken, may result in a Code of Conduct violation. Such actions include but are not limited to utilizing inappropriate means of communication, abusive emails directed toward staff, repeatedly submitting inappropriate material, and may occur on any of arXiv’s channels. The Code of Conduct does not address issues concerning the acceptance of research articles. Any issues concerning moderation can utilize the appeals process found at: (/help/moderation). 
+
+If a community member feels that a violation has occurred, the violation can be reported via the [Code of Conduct reporting form](https://cornell.ca1.qualtrics.com/jfe/form/SV_6FgHTrYs5JT30wZ). Please note that any communication received through this form is treated confidentially in keeping with arXiv’s privacy policies. 
+
+A committee formed of a combination of arXiv staff, leadership, and/or advisory board members will then determine whether or not a violation has occurred.
```
Comment (afromme) [inline help/policies/code_of_conduct_enforcement.md:11]: Suggestion: change "advisory board members" to "advisors"

File: help/policies/code_of_conduct_enforcement.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+If a community member feels that a violation has occurred, the violation can be reported via the [Code of Conduct reporting form](https://cornell.ca1.qualtrics.com/jfe/form/SV_6FgHTrYs5JT30wZ). Please note that any communication received through this form is treated confidentially in keeping with arXiv’s privacy policies. 
+
+A committee formed of a combination of arXiv staff, leadership, and/or advisory board members will then determine whether or not a violation has occurred. 
+
+If it is determined that a violation of the Code of Conduct has occurred, arXiv will take enforcement actions. The enforcement will depend on a number of factors, including the channel where the violation happened, whether it was directed at a specific individual or a group of people, if it is part of a pattern of repeated violations, and the severity of the violation. 
+
+The following are examples include but are not limited to the types of enforcement actions, which may be taken alone or in combination:
+
+* Email
```
Comment (afromme) [inline help/policies/code_of_conduct_enforcement.md:17]: Suggestion: "Email Notification"

File: help/policies/code_of_conduct_enforcement.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+If a community member feels that a violation has occurred, the violation can be reported via the [Code of Conduct reporting form](https://cornell.ca1.qualtrics.com/jfe/form/SV_6FgHTrYs5JT30wZ). Please note that any communication received through this form is treated confidentially in keeping with arXiv’s privacy policies. 
+
+A committee formed of a combination of arXiv staff, leadership, and/or advisory board members will then determine whether or not a violation has occurred. 
+
+If it is determined that a violation of the Code of Conduct has occurred, arXiv will take enforcement actions. The enforcement will depend on a number of factors, including the channel where the violation happened, whether it was directed at a specific individual or a group of people, if it is part of a pattern of repeated violations, and the severity of the violation. 
+
+The following are examples include but are not limited to the types of enforcement actions, which may be taken alone or in combination:
```
Comment (afromme) [inline help/policies/code_of_conduct_enforcement.md:15]: Suggestion to replace this sentence with: The following are examples of possible enforcement actions, which may be taken alone, in combination, or with additional measures not listed here.

**Review by afromme — APPROVED** [review]
Looks good. Proposed language changes are suggestions only.

File: help/policies/code_of_conduct_enforcement.md (hunk 3 above)
Comment (jimentwood) [reply help/policies/code_of_conduct_enforcement.md:17]: accepted

File: help/policies/code_of_conduct_enforcement.md (hunk 4 above)
Comment (jimentwood) [reply help/policies/code_of_conduct_enforcement.md:15]: sounds good

File: help/policies/code_of_conduct_enforcement.md (hunk 2 above)
Comment (jimentwood) [reply help/policies/code_of_conduct_enforcement.md:11]: accepted

File: help/policies/code_of_conduct_enforcement.md (hunk 1 above)
Comment (jimentwood) [reply help/policies/code_of_conduct_enforcement.md:5]: accepted

**Review by jweiskoff — COMMENTED** [review]
Why "Options" on the page name? What's the added value versus just saying "code of conduct enforcement"? I see that we called it that elsewhere as well, so I'm not wholesale against it...just curious?

**Comment by jimentwood** [comment]
I agree with @jweiskoff  comment that we can be more direct and drop the "options" from the title. This is done.

**Pull request merged**